meraki_wireless.tf

```diff
@@ -394,13 +394,14 @@ locals {
     for domain in try(local.meraki.domains, []) : [
       for organization in try(domain.organizations, []) : [
         for network in try(organization.networks, []) : [
-          for wireless_ssid in try(network.wireless_ssids, []) : [
+          for wireless_ssid in try(network.wireless.ssids, []) : [
             for identity_psk in try(wireless_ssid.identity_psks, []) : {
-              network_id = meraki_network.network["${organization.name}/${network.name}"].id
-              number     = meraki_wireless_ssid.net_wireless_ssids["${organization.name}/${network.name}/ssids/${wireless_ssid.name}"].number
-              data       = try(identity_psk, null)
+              network_id      = meraki_network.network["${organization.name}/${network.name}"].id
+              number          = meraki_wireless_ssid.net_wireless_ssids["${organization.name}/${network.name}/ssids/${wireless_ssid.name}"].number
+              data            = try(identity_psk, null)
+              group_policy_id = meraki_network_group_policy.net_group_policies["${organization.name}/${network.name}/${identity_psk.group_policy_name}"].id
```

Comment:
All of the changes from this PR except adding the `group_policy_id` mapping are already done as part of the refactor in `main`. After the rebase, this would have to use `meraki_network_group_policy.networks_group_policies`, `format()` and add `domain.name` to the lookup key.